### ISSUE TITLE
Release 2024112501-public Images Might be Wrong

### ISSUE THREAD

**Issue by GiovanniYuriMita** [hystax/optscale#480]
**Describe the bug**
Hey guys, I'm here again.

This time, I noticed that the latest release ([2024112501-public](https://github.com/hystax/optscale/releases/tag/2024112501-public)) might not be built correctly -- When I deployed it, I saw that all the services that were meant to have new features, such as ngui, rest_api and herald, stayed exactly the same as before.

I created a new EC2 and did all the deployment again, from zero. The results were the same. I then cloned the repository with the latest changes and started ngui in local environment since I knew I could test my theory using the email verification workflow and, comparing the local ngui with the 2024112501-public image, I instantly noticed that the cluster one didn't have the email verification workflow. Besides that, when I invoked verify_email using postman or the local ngui, it returned 404 as this image of rest_api also weren't updated yet.

P.S.: I also tried using the docker hub images directly (i.e.: hystax/rest_api:2024112501-public) since I wanted to rule out a possible issue with the deployment method, but got the same results.

I think the whole release might be compiled in a repo that didn't fetch the latest commits

**To Reproduce**

```
sudo su
apt update; apt install python3-pip sshpass git python3-virtualenv python3.9 python3.9-venv awscli -y
```

```
git clone https://github.com/hystax/optscale.git
cd optscale/optscale-deploy
```
```
virtualenv -p python3.9 venv
source venv/bin/activate
pip install -r requirements.txt
```

```
ansible-playbook -e "ansible_connection=local ansible_user=ubuntu" -i "<ip>," ansible/k8s-master.yaml
```

```
./runkube.py --with-elk -o overlay/user_template.yml -- optscale-deployment 2024112501-public
```

**Expected behavior**
Upon deploying the 2024112501-public release, all services (ngui, rest_api, herald, etc.) should include the latest features as per the recent commits

**Screenshots**
![image](https://github.com/user-attachments/assets/42c09cd0-d533-4b57-98fb-c2b74722bcbb)
![image](https://github.com/user-attachments/assets/4bb118f0-960b-4ecb-a566-8d19f4dec606)

**Fun fact**
I spent almost 3 weeks trying to implement an email-verification workflow on OptScale, and after LOTS of struggles, I finally did it yesterday and it worked perfectly. It was beautiful! Then I noticed you did it about 4 days ago in this latest release 🫠

At least this experience has deepened my understanding on OptScale's structure and how to implement new ideas 🙂

**Comment by stanfra** [comment]
Hi @GiovanniYuriMita, Email verification is not included in the release 2024112501-public, the feature was merged later, on November 28th (https://github.com/hystax/optscale/commit/61300f60d00fb85389753e96ff0ccee5abdd8f5b). If you want to deploy the feature to a cluster, you can build images from the latest integration or from the commit I specified locally, using build.sh script in optscale repo and then start cluster with` --no-pull` flag and `local` as version

**Comment by GiovanniYuriMita** [comment]
Hi @stanfra,

Thank you for the explanation! In fact, I didn't compare the commit date with the release date.

I appreciate your patience!
